platform/masterfs/src/org/netbeans/modules/masterfs/filebasedfs/children/ChildrenCache.java

```diff
@@ -26,8 +26,8 @@
 import java.util.Set;
 
 public interface ChildrenCache {
-    Integer ADDED_CHILD = new Integer(0);
-    Integer REMOVED_CHILD = new Integer(1);
+    Integer ADDED_CHILD = 0;
+    Integer REMOVED_CHILD = 1;
```

Comment:
switched all usages which compared with `==` to `equals`

usages in `FolderObj` and `ChildrenSupportTest`